protocol-2.5.md

```diff
@@ -25,7 +28,7 @@ differently.
 | report analog pin     | 0xC0    | pin #        | disable/enable(0/1) | - n/a -         |
 | report digital port   | 0xD0    | port         | disable/enable(0/1) | - n/a -         |
 |                       |         |              |                     |                 |
-| start sysex           | 0xF0    |              |                     |                 |
+| start sysex           | 0xF0    | sub-command  | ...                 |                 |
```

Comment:
Yes, that was a typo on my part.  I was trying to show that the first byte after the StartSysex was a sub-command and that arbitrary data followed, but I put them in the wrong column.

Thx.

Doug

> On Oct 31, 2015, at 1:34 PM, Jeff Hoefs <notifications@github.com> wrote:
> 
> In protocol-2.5.md <https://github.com/firmata/protocol/pull/40#discussion_r43575899>:
> 
> > @@ -25,7 +28,7 @@ differently.
> >  | report analog pin     | 0xC0    | pin #        | disable/enable(0/1) | - n/a -         |
> >  | report digital port   | 0xD0    | port         | disable/enable(0/1) | - n/a -         |
> >  |                       |         |              |                     |                 |
> > -| start sysex           | 0xF0    |              |                     |                 |
> > +| start sysex           | 0xF0    | sub-command  | ...                 |                 |
> These are not sub-commands. For the first 4 rows, the 2nd column is combined with the first column just as in the midi protocol. Just need to remove the "sub-command" string and the ellipse from the 3rd column both in the 2.4 version and the 2.5 version. Could combine the command and midi channel columns to better indicate this. So the header and column 1 would be:
> 
> | type                             | command + midi channel  | first byte              | second byte     |
> | analog I/O message    | 0xE0 + pin #                      | LSB(bits 0-6)       | MSB(bits 7-13)  |
> —
> Reply to this email directly or view it on GitHub <https://github.com/firmata/protocol/pull/40/files#r43575899>.
>